tests/models_test.py

```diff
@@ -91,3 +94,42 @@ def test_no_top_variable_shape_4(app, last_feature_dim):
     input_shape = (None, None, 4)
     model = app(weights=None, include_top=False, input_shape=input_shape)
     assert model.output_shape == (None, None, None, last_feature_dim)
+
+
+@parametrize
+def test_model_summary(app, last_feature_dim, capsys, snapshot):
+    input_tensor = keras.layers.Input(shape=(224, 224, 3))
+    model = app(weights=None, input_tensor=input_tensor)
+    lq.models.summary(model)
+    out, err = capsys.readouterr()
```

Comment:
I'd prefer to snapshot `model.to_json()` here since this will be invariant on non-functional changes and will work across different larq versions.

Reply:
This would be nice for testing purposes as we're not affected by changes in formatting, but does make the files unreadable. For now I do think JSON is the best solution, if we explicitly want to have the formatted summaries somewhere we should probably create a script that adds them to the docs.